src/modules/create-market/components/create-market-form-resolution/create-market-form-resolution.styles.less

```diff
@@ -42,6 +44,18 @@ li.CreateMarketResolution__designated-reporter-specific {
       z-index: 2;
     }
 
+    .SingleDatePickerInput {
```

Comment:
Left: Seadragon
Right: Branch

Lack of contrast gives the appearance the dropdown is wider than the input box. I cannot find a mockup of this so I'm calling it out here.

<img width="1067" alt="screen shot 2018-02-26 at 4 08 10 pm" src="https://user-images.githubusercontent.com/328965/36703203-622c96da-1b0f-11e8-9bd5-0380674ed658.png">

Reply:
Yeah, I was torn about that too. I'm going to remove the border on focus.